README.md

```diff
@@ -47,11 +47,11 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo]()
+- [Live Demo](https://richie1988.github.io/SuperMario/)
```

Comment:
- [ ] your live link is blank, kindly host your app on Netlify or render to avoid such
![Screenshot from 2023-10-18 19-52-42](https://github.com/richie1988/SuperMario/assets/66097371/b532e2fd-84cc-45fe-9796-9931aab2d399)